client/src/main.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import App from "./App.tsx";
+import { Provider } from "react-redux";
+import { store } from "./state/store.ts";
```

Comment:
nie powinieneś musieć podawać rozszerzenia importując normalne moduły
```suggestion
import { store } from "./state/store";
```
także to powinno też zadziałać. Jeśli nie, to coś jest skopane w konfigu